mkosi.md

```diff
@@ -1143,6 +1143,14 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
   the same location, suffixed with `.pub` (as done by `ssh-keygen`). If this
   option is not present, `mkosi` generates a new key pair automatically.
 
+`SshAgent=`, `--ssh-agent=`
+
+: If specified as a path, use the given socket to connect to the ssh agent when
+  building an image and when connecting via `mkosi ssh` instead of hard-coding
+  a key. If specified as `true`, `$SSH_AUTH_SOCK` will be parsed instead (hint:
+  use `sudo` with `-E`). The available public keys will be installed at build
+  time from the output of `ssh-add -L`.
```

Comment:
That last sentence is awkward. Maybe "The keys listed by `ssh-add -L` will be installed as authorized_keys in the built image." ?